test/system/500-networking.bats

```diff
@@ -729,10 +738,22 @@ nameserver 8.8.8.8" "nameserver order is correct"
         cid="$output"
         for port in $(seq $port $end_port); do
             run_podman exec -d $cid nc -l -p $port -e /bin/cat
-            # -w 1 adds a 1 second timeout. For some reason, ubuntu's ncat
-            # doesn't close the connection on EOF, and other options to
-            # change this are not portable across distros. -w seems to work.
-            run nc -w 1 127.0.0.1 $port <<<$random
+
+            # we have to rety ncat as it can flake as we exec in the background so nc -l
+            # might not have bound the port yet, retry seems simpler than checking if the
+            # port is bound in the container, https://github.com/containers/podman/issues/21561.
+            retries=5
+            while [[ $retries -gt 0 ]]; do
+                # -w 1 adds a 1 second timeout. For some reason, ubuntu's ncat
+                # doesn't close the connection on EOF, and other options to
```

Comment:
I wonder if this is still relevant, given that we no longer test on ubuntu? That special-case code dates from March 2022, #13646

Reply:
We would need to test that, although it makes sure nc would time out and not hang forever.